localclient/terminal_player.go

```diff
@@ -264,6 +268,17 @@ func getGameIDAndBody(c *gin.Context, defBody string) (model.GameID, string, err
 	return model.GameID(n), msg, nil
 }
 
+func (tc *terminalClient) makeJSONBodiedRequest(method, apiURL string, v interface{}) ([]byte, error) {
+	b, err := json.Marshal(v)
+	if err != nil {
+		return nil, err
+	}
+	header := http.Header{
+		`Content-Type`: []string{`application/json`},
```

Comment:
🔮  If the content `b` is empty, do we still wanna set this as "json" content-type?